### ISSUE TITLE
thread 'main' panicked at 'IndexMap: key not found

### ISSUE THREAD

**Issue by AKikhaev** [hikari-no-yume/touchHLE#83]
Hello. I tried to  run old 32 bit game Trainyard. 

My version mac os Ventura 13.2.1, intel mac book pro 2017

When I try to run it or get information i have the same error:
./touchHLE Trainyard.ipa --info

```
 ✘  ~/Desktop/touchHLE_v0.1.2_macOS_x86_64  RUST_BACKTRACE=1 ./touchHLE Trainyard.ipa --info
touchHLE v0.1.2 — https://touchhle.org/

thread 'main' panicked at 'IndexMap: key not found', src/bundle.rs:38:27
stack backtrace:
   0: _rust_begin_unwind
   1: core::panicking::panic_fmt
   2: core::panicking::panic_display
   3: core::panicking::panic_str
   4: core::option::expect_failed
   5: <plist::dictionary::Dictionary as core::ops::index::Index<&str>>::index
   6: touchHLE::main
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
 ✘  ~/Desktop/touchHLE_v0.1.2_macOS_x86_64  RUST_BACKTRACE=full ./touchHLE Trainyard.ipa --info
touchHLE v0.1.2 — https://touchhle.org/

thread 'main' panicked at 'IndexMap: key not found', src/bundle.rs:38:27
stack backtrace:
   0:        0x102a07cf2 - <std::sys_common::backtrace::_print::DisplayBacktrace as core::fmt::Display>::fmt::hdc69a6f447628e71
   1:        0x102a2321a - core::fmt::write::hb9e764fa47ae8444
   2:        0x102a0522c - std::io::Write::write_fmt::h8fc98987ed860a54
   3:        0x102a07aba - std::sys_common::backtrace::print::h488fe4c0b1fb9d50
   4:        0x102a09186 - std::panicking::default_hook::{{closure}}::h5618ea3156b8b833
   5:        0x102a08ed7 - std::panicking::default_hook::h0421c26a8a92801c
   6:        0x102a098cd - std::panicking::rust_panic_with_hook::h57383cd32463c250
   7:        0x102a09683 - std::panicking::begin_panic_handler::{{closure}}::h1d1f7305cfe67fdd
   8:        0x102a08188 - std::sys_common::backtrace::__rust_end_short_backtrace::hd8e12e82ff026bae
   9:        0x102a0934d - _rust_begin_unwind
  10:        0x102a3aab3 - core::panicking::panic_fmt::h7894cd1015cfee41
  11:        0x102a2114b - core::panicking::panic_display::h3f1a5f08bdfae356
  12:        0x102a210fc - core::panicking::panic_str::h2c641beb2ba2ff86
  13:        0x102a3aa79 - core::option::expect_failed::hb02180faedd071ab
  14:        0x10293a549 - <plist::dictionary::Dictionary as core::ops::index::Index<&str>>::index::h8faa689fa9db6ad7
  15:        0x1024fd72e - touchHLE::main::hd1bfddb36db96448
  16:        0x1024e257b - std::sys_common::backtrace::__rust_begin_short_backtrace::h5fe70793acba79a0
  17:        0x10250bde5 - std::rt::lang_start::{{closure}}::h9145ee28bf22adcc
  18:        0x102a008a4 - std::rt::lang_start_internal::h9ca2efac34d80f78
  19:        0x1025029cc - _main
  20:     0x7ff801c9e310 - <unknown>
```

This  IPA file I have download from my own app store account using iMazing tool, (Library feature).

Should I use this tool for this? Is it works on intel?

**Comment by hikari-no-yume** [comment]
> Please bear in mind that there are infinitely many apps that do not work in touchHLE right now, so please don't open issues about apps that aren't in the supported list

([_Contributing to touchHLE_](https://github.com/hikari-no-yume/touchHLE/blob/trunk/CONTRIBUTING.md))

**Comment by hikari-no-yume** [comment]
The particular thing that's causing this to crash is fixed in the next version (c1c746202b499f1b1c9c3b2acd04bf7de7968637), but there will probably be many other things that also stop it from working.
